frontend/src/app/modules/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="no-found">
```

Comment:
Why do we need this component if we have same in _frontend/src/app/shared/components/not-found/not-found.component.html_ ?